process/scToProtocol/stakingToPeer.go

```diff
@@ -226,13 +226,6 @@ func (stp *stakingToPeer) updatePeerStateV1(
 		}
 	}
 
-	if !bytes.Equal(account.GetBLSPublicKey(), blsPubKey) {
-		err = account.SetBLSPublicKey(blsPubKey)
```

Comment:
don't we need to use `GetPeerAccountAndReturnIfKnow` here also? or is the operations in `getPeerAccount` enough here?